source/read/count-documents.txt

```diff
@@ -0,0 +1,221 @@
+.. _java-rs-count:
+
+===============
+Count Documents
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation, code example
+
+Overview
+---------
+
+In this guide, you can learn how to retrieve an accurate and estimated count of the
```

Comment:
Q: do we use "accurate" elsewhere in our docs to refer to this kind of count? just asking because [pushes glasses up nose] technically, an estimate can also be highly accurate

Reply:
This is the wording we used in pymongo. I can change it to "an exact count"

Reply:
I do think "exact count" is better!